types/node/v16/net.d.ts

```diff
@@ -18,8 +18,8 @@ declare module "net" {
     import * as dns from "node:dns";
     type LookupFunction = (
         hostname: string,
-        options: dns.LookupOneOptions,
-        callback: (err: NodeJS.ErrnoException | null, address: string, family: number) => void,
+        options: dns.LookupOptions,
+        callback: (err: NodeJS.ErrnoException | null, address: string | dns.LookupAddress[], family?: number) => void,
```

Comment:
I'm not sure this correct. From what I can see, "lookup multiple" only landed in Node 18+ with the advent of auto-select-family

Specifically Node 18.13:
- Release notes: https://nodejs.org/en/blog/release/v18.13.0
- PR: https://github.com/nodejs/node/pull/44731

For reference, this is the Node 16 code: https://github.com/nodejs/node/blob/76f9a7d27f5ae9852b9ffc67a7dec1af503e0813/lib/net.js#L1107-L1195

Reply:
You're right, thanks for catching that! I assumed `net.LookupFunction` and `dns.lookup` needed to match but that's not the case prior to Node 18, I amended the commit to leave `v16` as it was.